src/wad/level.rs

```diff
@@ -152,12 +150,14 @@ impl Level {
         let mut min_light = sector.light;
         let sector_id = self.sector_id(sector);
         for line in self.linedefs.iter() {
-            let (left, right) =
-                (match self.left_sidedef(line) {
-                    Some(l) => l.sector, None => continue,
-                 }, match self.right_sidedef(line) {
-                    Some(r) => r.sector, None => continue,
-                 });
+            let (left, right) = (match self.left_sidedef(line) {
+                Some(l) => l.sector,
```

Comment:
We tend to indent match arms to the current block indent + 1 tab, so this is technically correct, although in this instance it looks very bad. I think we are unlikely to do better here, so probably better to use a local variable (or #[rustfmt_skip]).